code/datums/mood_events/generic_negative_events.dm

```diff
@@ -80,15 +80,6 @@
 	mood_change = -2
 	timeout = 1200
 
-/datum/mood_event/table/add_effects()
-	if(ishuman(owner))
-		var/mob/living/carbon/human/H = owner
-		if(iscatperson(H))
-			H.dna.species.start_wagging_tail(H)
-			addtimer(CALLBACK(H.dna.species, /datum/species.proc/stop_wagging_tail, H), 30)
-			description =  "<span class='nicegreen'>They want to play on the table!</span>\n"
```

Comment:
this is a good feature please unremove this

Reply:
remove tabling

Reply:
No